std_msgs/Header header

FixationDataPoint[] fixations
